### ISSUE TITLE
RTL language support

### ISSUE THREAD

**Issue by ddugovic** [WandererXII/lishogi#658]
> hello lishogi!
> 
> I am here to suggest you to add Farsi language.
> I will personally take full responsibility for its translation and translation into Farsi.
> 
> Thanks for reading.

https://lishogi.org/forum/general-shogi-discussion/blog-ZBxnNBAAACIA599h#2

**Comment by WandererXII** [comment]
Can't we just add Farsi without RTL support, because honestly it looks like a real pain to do - (https://github.com/lichess-org/lila/pull/11019, https://github.com/lichess-org/lila/search?q=rtl&type=issues). Not being a speaker of RTL languages, I can't really say, how necessary it is...
But we currently offer for example Arabic without RTL..

**Comment by ddugovic** [comment]
Good point; I'll ask the person proposing Farsi for clarification (and if they don't know, I'll ask around).
